src/applications/pensions/config/chapters/06-additional-information/supportingDocuments.js

```diff
@@ -160,7 +161,7 @@ function Description({ formData }) {
 
 export default {
   uiSchema: {
-    'ui:title': 'Supporting documents',
+    ...titleUI('Supporting documents'),
     'ui:description': Description,
```

Comment:
move to titleUI arg

Reply:
This description includes multiple lists and components that may be out of the best use case of a 'titleUI' description.